tests/contracts/testContracts.nim

```diff
@@ -38,10 +38,10 @@ ethersuite "Marketplace contracts":
     request.client = await client.getAddress()
 
     switchAccount(client)
-    await token.approve(marketplace.address, request.price)
+    discard await token.approve(marketplace.address, request.price)
```

Comment:
probably want `asyncSpawn`, otherwise you're swallowing exceptions, which is fun to debug.

Reply:
I'm not discarding the future, I'm discarding the transaction receipt, after awaiting the future :smile:

Reply:
ah, you're right :)